### DISCUSSION TITLE
How to send files and data using POST requests?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [aio-libs/aiohttp#7042] [Q&A] [answered]
How do I rewrite this
```python
 files = {'file1': open(fileName,'rb')}
        r = requests.post(requestURL, files=files, data=values)
```
        
using aiohttp and I suppose asynchronous file read?

**Comment by Dreamsorcerer** [comment, marked as answer]
Doesn't this answer your question?
https://docs.aiohttp.org/en/stable/client_quickstart.html#post-a-multipart-encoded-file

I literally just searched for "file" on the client page..

**Reply by martinenkoEduard** [reply]
Thanks!)
